app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Why is there a mailer layout? Can this also be deleted?

Reply:
deleted